City-of-Helsinki/berth-reservations
# VEN-1015 | Default sorting for leases

## Description :sparkles:
* By default, order the leases by decreasing start/end date
* Few tweaks for lease Django admin

## Issues :bug:
### Closes :no_good_woman:
**[VEN-1015](https://helsinkisolutionoffice.atlassian.net/browse/VEN-1015):** Sort Laskuhistoria from newest to oldest

## Testing :alembic:
### Manual testing :construction_worker_man:
Fetch the leases and check that the start date of the first results is later (or equal) than the later ones
```graphql
query Leases {
  winterStorageLeases {
    edges {
      node {
        startDate
        endDate
      }
    }
  }
}
```

```graphql
query Leases {
  berthLeases {
    edges {
      node {
        startDate
        endDate
      }
    }
  }
}

```

## Screenshots :camera_flash:
Few more sorting options on `WinterStorageLease` page
<img width="1134" alt="image" src="https://user-images.githubusercontent.com/15201480/129878870-d0282e0c-b4e4-4743-b3a6-ceb148f728e1.png">
